api/types/state/state.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package containerd.v1.types;
+
+// At the moment we have same State type for Container and Process
+enum State {
```

Comment:
These (`State`, `Process`, `User`) should all be in `process`. They are all related to processes.